backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/cgen/CBridgeGen.kt

```diff
@@ -261,6 +262,10 @@ private fun <R> KotlinToCCallBuilder.handleArgumentForVarargParameter(
             } else {
                 stubs.throwCompilerError(initializer, "unexpected initializer")
             }
+        } else if (variable is IrValueParameter && CallableReferenceLowering.isLoweredCallableReference(variable)) {
+            val location = variable.parent // Parameter itself has incorrect location.
+            val kind = if (this.isObjCMethod) "Objective-C methods" else "C functions"
+            stubs.reportError(location, "callable references to variadic $kind are not supported")
```

Comment:
Why plural? It seems there's only one reference here.

Reply:
According to popular convention for Kotlin compiler error messages.
See e.g. a lot of examples here: https://github.com/JetBrains/kotlin/blob/056c3c95bb3e02a6c131a7c80d1afa6dc6f4d050/compiler/frontend/src/org/jetbrains/kotlin/diagnostics/rendering/DefaultErrorMessages.java#L550